src/ert/scheduler/openpbs_driver.py

```diff
@@ -142,16 +142,35 @@ def __init__(
         self._finished_job_ids: Set[str] = set()
         self._finished_iens: Set[int] = set()
 
-        self._resources = self._resource_strings()
+        if self._num_nodes is not None and self._num_nodes > 1:
+            logger.warning(
+                "OpenPBSDriver initialized with num_nodes > 1, "
+                "this behaviour is deprecated and will be removed"
+            )
+
+        if self._num_cpus_per_node is not None and self._num_cpus_per_node > 1:
+            logger.warning(
+                "OpenPBSDriver initialized with num_cpus_per_node, "
+                "this behaviour is deprecated and will be removed. "
+                "Use NUM_CPU in the config instead."
+            )
 
-    def _resource_strings(self) -> List[str]:
+    def _build_resource_string(self, num_cpu: int = 1) -> List[str]:
         resource_specifiers: List[str] = []
 
         cpu_resources: List[str] = []
         if self._num_nodes is not None:
             cpu_resources += [f"select={self._num_nodes}"]
         if self._num_cpus_per_node is not None:
-            cpu_resources += [f"ncpus={self._num_cpus_per_node}"]
+            num_nodes = self._num_nodes or 1
+            if num_cpu != self._num_cpus_per_node * num_nodes:
+                raise ValueError(
+                    f"NUM_CPUS_PER_NODE ({self._num_cpus_per_node}) must be equal "
+                    f"to NUM_CPU ({num_cpu}). "
+                    "Please remove NUM_CPUS_PER_NODE from the configuration"
+                )
```

Comment:
Isn't this part already checked for during config validation?

Reply:
It is, but in case rogue developers would use the driver independently it should also fail i suppose.

Reply:
Yes, I agree